### PR TITLE
Milestone 1.1.0

### PR DESCRIPTION
**Pull request by Scrum**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ module.exports = (options = {}) => tree => {
   options.tags = options.tags || ['fetch', 'remote']
   options.attribute = options.attribute || 'url'
   options.got = options.got || {}
+  options.stayTag = options.stayTag || false
```
Comment (cossssmin) [inline lib/index.js:12]: Can we please rename this option to `preserveTag` instead of `stayTag`?

Comment (Scrum) [reply lib/index.js]: done

File: README.md [hunk 2]
```diff
@@ -135,6 +135,56 @@ posthtml()
...
+const pf = require('posthtml-fetch')
+
+posthtml()
+  .use(pf({
+    preserveTag: true
+  }))
+  .process('<fetch url="https://example.test">{{ response }}</fetch>')
+  .then(result => console.log(result.html))
+```
+
```
Comment (cossssmin) [inline README.md:155]: Can you please also show what the resulting HTML will be? Like:

Result:

```html
... the resulting HTML 
```

Comment (Scrum) [reply README.md:164]: done

**Pull request merged**